### ISSUE TITLE
Create an API key with the action with `*` for `indexes`, `settings`, `tasks`, `dumps`, `stats`

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch#2560]
Checked with @gmourier, we can let the possibility to the users to set keys with the following actions
- `indexes.*`: all the index actions are available, so `indexes.create`, `indexes.get`, `indexes.update`, and `indexes.delete`
- `settings.*`: all the settings actions are available, so `settings.get`, `settings.update`
- `tasks.*`: all the tasks actions are available, it means only `tasks.get` so far.
- `dumps.*`: all the tasks actions are available, it means only `dumps.get` so far.
- `stats.*`: all the tasks actions are available, it means only `stats.get` so far.

This possibility has already been done for documents (`documents.*`), check out [this PR](https://github.com/meilisearch/meilisearch/pull/2525#issuecomment-1168791835) as a clue for the implementation

## For Meili people

Since we let this issue (so far) to the community, we don't know yet in which release this new feature will be available (at minimum v0.29.0, not before). The specifications already have been changed according to this, see https://github.com/meilisearch/specifications/blob/develop/text/0085-api-keys.md

**Comment by curquiza** [comment]
Thanks you @janithpet ❤️ I've just updated the issue description with more expectations 😇

**Comment by curquiza** [comment]
pinging @meilisearch/integration-team and @meilisearch/docs-team FYI

**Comment by curquiza** [comment]
Hello everyone watching this issue!

We have just released [v0.29.0rc1](https://github.com/meilisearch/meilisearch/releases/tag/v0.29.0rc1), which is a release candidate of v0.29.0 🔥 
It means you can test the just-introduced `*` helper when using the `actions` of tenant tokens! [Binaries](https://github.com/meilisearch/meilisearch/releases/tag/v0.29.0rc1) are attached to the release, or you can you the docker image:
```bash
docker run -it --rm \
    -p 7700:7700 \
    getmeili/meilisearch:v0.29.0rc1
```

Let us know about any bugs or feedback! 😄 It would be really helpful.

FYI, the official v0.29.0 release will be available on 3rd October.
